Rapportering/DSN/format_data.py

```diff
@@ -109,6 +192,36 @@ def _categorize_exams_according_to_ssm(data: pd.DataFrame, modality: str) -> pd.
         for exam_name, exam_group_values in exam_group.items():
             if not exam_group_values:
                 continue
-            data.loc[data[grouping_column].isin(exam_group_values), [OUTPUT_COL_EXAM]] = exam_name
+            if CHILD_EXAM_PREFIX in exam_name:
```

Comment:
Bara för att göra det ännu mer "fool proof" så hade jag nog gjort denna koll på att `exam_name.startswith(CHILD_EXAM_PREFIX)` Då råkar man inte ut för att någon okänd person har råkat stoppa in den strängen även senare i ett protokoll. Det händer nog inte här men det är lite "renare" :)